### DISCUSSION TITLE
Do we have any repository for sample modules

### DISCUSSION THREAD

**Discussion by eshirke** [SAP/terraform-provider-btp#981] [General]
Do we have any repository for sample modules for BTP

**Comment by lechnerc77** [comment]
There is no official repository for modules from SAP as we do not provide modules as of today.

Community modules are hosted in the in individual repositories of the maintainers see e.g.,

- https://registry.terraform.io/modules/aydin-ozcan/sap-btp-entitlements/btp/latest
- https://registry.terraform.io/modules/codeyogi911/hana-cloud/sap/latest

**Reply by eshirke** [reply]
thanks for details, it really helps
